conf/routes

```diff
@@ -113,3 +113,100 @@ DELETE /admin/v1/users          @org.sagebionetworks.bridge.play.controllers.Use
+POST   /v3/surveyresponses/:guid                                        @org.sagebionetworks.bridge.play.controllers.SurveyResponseController.appendSurveyAnswers(guid: String)
+POST   /v3/surveyresponses/:surveyGuid/revisions/:createdOn             @org.sagebionetworks.bridge.play.controllers.SurveyResponseController.createSurveyResponse(surveyGuid: String, createdOn: String)
+POST   /v3/surveyresponses/:surveyGuid/revisions/:createdOn/:identifier @org.sagebionetworks.bridge.play.controllers.SurveyResponseController.createSurveyResponseWithIdentifier(surveyGuid: String, createdOn: String, identifier: String)
+
+# Study Consents & Signatures
+GET    /v3/consents                     @org.sagebionetworks.bridge.play.controllers.StudyConsentController.getAllConsents
+GET    /v3/consents/recent              @org.sagebionetworks.bridge.play.controllers.StudyConsentController.getMostRecentConsent
+GET    /v3/consents/active              @org.sagebionetworks.bridge.play.controllers.StudyConsentController.getActiveConsent
+GET    /v3/consents/:timestamp          @org.sagebionetworks.bridge.play.controllers.StudyConsentController.getConsent(timestamp: String)
+POST   /v3/consents/:timestamp/activate @org.sagebionetworks.bridge.play.controllers.StudyConsentController.setActiveConsent(timestamp: String)
+POST   /v3/consents                     @org.sagebionetworks.bridge.play.controllers.StudyConsentController.addConsent
+GET    /v3/consents/signature           @org.sagebionetworks.bridge.play.controllers.ConsentController.getConsentSignature
+POST   /v3/consents/signature           @org.sagebionetworks.bridge.play.controllers.ConsentController.giveV2
+POST   /v3/consents/signature/email     @org.sagebionetworks.bridge.play.controllers.ConsentController.emailCopy
+
+# Upload Schemas
+GET    /v3/uploadschemas                               @org.sagebionetworks.bridge.play.controllers.UploadSchemaController.getUploadSchemasForStudy
+POST   /v3/uploadschemas                               @org.sagebionetworks.bridge.play.controllers.UploadSchemaController.createOrUpdateUploadSchema
+GET    /v3/uploadschemas/:schemaId                     @org.sagebionetworks.bridge.play.controllers.UploadSchemaController.getUploadSchema(schemaId: String)
+DELETE /v3/uploadschemas/:schemaId                     @org.sagebionetworks.bridge.play.controllers.UploadSchemaController.deleteUploadSchemaById(schemaId: String)
+DELETE /v3/uploadschemas/:schemaId/revisions/:revision @org.sagebionetworks.bridge.play.controllers.UploadSchemaController.deleteUploadSchemaByIdAndRev(schemaId: String, revision: Int)
+
+# Uploads
+POST   /v3/uploads              @org.sagebionetworks.bridge.play.controllers.UploadController.upload
+GET    /v3/uploads/:id          @org.sagebionetworks.bridge.play.controllers.UploadController.getValidationStatus(id: String)
```

Comment:
I feel like GET /v3/uploads/:id implies we're getting the upload itself. I prefer GET /v3/uploads/:id/status to make it clear we're only getting the upload validation status.